spec/system/nav_bar_system_spec.rb

```diff
@@ -45,4 +45,22 @@
 
     logout
   end
+
+  context 'Copy CCLI Login' do
+    it 'shows Copy CCLI Login button' do
+      login_as_user(:admin)
+      dropdown_menus = all('pzsh-menu-dropdown', text: users(:admin).givenname).to_a
+      dropdown_menus.second.click
+      expect(find('pzsh-menu-dropdown-item', text: 'Copy CCLI Login')).to be_present
+    end
+
+    it 'copies ccli login command to clipboard' do
+      login_as_user(:admin)
+      dropdown_menus = all('pzsh-menu-dropdown', text: users(:admin).givenname).to_a
+      dropdown_menus.second.click
+      find('pzsh-menu-dropdown-item', text: 'Copy CCLI Login').click
+      expect(find('span.message', text: 'CCLI Login command was copied!')).to be_present
```

Comment:
Have you checked if there is a way to see what was copied to the clipboard?
This way we only check if we have a success message however we don't check the clipboard functionality.
Would be nice to have :smile_cat:

Reply:
Unfortunately i can't figure out a way to read the clipboard content.
I have looked at the clipboard API, but i can't get that to work.
https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API

Reply:
Alright that's fine, was worth a shot :grin: